src/main/java/org/sagebionetworks/bridge/spring/controllers/UserManagementController.java

```diff
@@ -71,6 +71,14 @@ public JsonNode signInForAdmin() throws Exception {
         return UserSessionInfo.toJSON(session);
     }
     
+    /**
+     * This turned out to be useful... so useful we're opening it up to all administrative
+     * users.
+     * 
+     * @see org.sagebionetworks.bridge.spring.controllersAuthenticationController#changeStudy 
+     * @throws Exception
+     */
+    @Deprecated
```

Comment:
AuthenticationController.changeStudy requires synapseUserId. This API does not.

Reply:
That's a good point. I moved a couple of checks around and now it is not required for this role, and updated the relevant changeStudy test to verify.